examples/02_fit_predict_plot_employee_salaries.py

```diff
@@ -16,23 +16,24 @@
 #
 # We first download the dataset:
 from dirty_cat.datasets import fetch_employee_salaries
-employee_salaries = fetch_employee_salaries()
-print(employee_salaries['DESCR'])
+
+info = fetch_employee_salaries()
+print(info['description'])
 
 
 ################################################################################
 # Then we load it:
 import pandas as pd
```

Comment:
I know that it's ugly, but I would rather that you:
1. print the info['read_csv_kwargs']
2. pass them explicitly to read_csv
3. add a comment saying that such operation is feasible with the syntax ...**info['read_csv_args']

The reason is that I am trying to reach out to people who do not understand this syntax